documentation/book/getting-started.adoc

```diff
@@ -12,21 +12,24 @@ include::common/attributes.adoc[]
 [id='getting-started-{context}']
 = Getting started
 
-{ProductLongName} runs on {ProductPlatformName}.
-{ProductName} supports
-ifdef::Kubernetes[{KubernetesLongName} {KubernetesVersion} or]
+{ProductName} runs on
+ifdef::Kubernetes[{KubernetesLongName} {KubernetesVersion} and]
 {OpenShiftLongName} {OpenShiftVersion}.
-{ProductName} works on all kinds of clusters - from public and private clouds down to local deployments intended for development.
+It can be used on clusters of public and private clouds, local deployments intended for development, and everything in between.
```

Comment:
Since I already opened it ... _clusters of public and private clouds_ sounds very strange to me. It could be even interpreted as stretch cluster between private and public clouds which might be challenging. I think it would work better as _... on cluster running in public and private clouds, ..._.